submission/ci_validation.py

```diff
@@ -0,0 +1,27 @@
+from pathlib import Path
+import os
+from byudml import __imputer_path__, __metafeature_path__
+import sys
+sys.path.append(".")
+from submission.utils import get_new_d3m_path
+
+byu_dir = get_new_d3m_path()
+print("##### Validating json files with D3M Validation #####")
+
+primitive_jsons = Path(byu_dir).glob('**/primitive.json')
+# for filename in primitive_jsons:
+#     # strip the init file path off, since we are changing directories
+#     filename_real = "/".join(str(filename).split("/")[2:])
+#     print("Validating primitive:", filename_real)
+#     os.system("cd submission/primitives/; python3 run_validation.py {}; cd ../..".format(filename_real))  
+
+# check pipelines and meta files
+for filename in Path(byu_dir).glob('**/*.json'):
+    if filename in primitive_jsons:
```

Comment:
I wonder if this is not working

Reply:
Apparently glob uses a PosixPath and so it isn't a string. That should be fixed now